doc/nrf/gs_modifying.rst

```diff
@@ -242,13 +257,15 @@ If you work with SES, the :file:`prj.conf` file is read when you open a project.
 The file will be reloaded when CMake re-runs.
 This will happen automatically when the application is rebuilt, but can also be requested manually by using the :guilabel:`Project` > :guilabel:`Run CMake...` option.
 
-.. _configuring_vsc:
+If you work with |VSC|, you can use one of the following options:
 
-Configuring in the VS Code extension
-====================================
+* Select an extra Kconfig fragment file when `Building an application`_.
+* Edit the Kconfig options using the `nRF Kconfig`_ extension.
+  To save changes permanently to the :file:`prj.conf` file, click the :guilabel:`Save to file` button.
```

Comment:
Is "permanently" the right word to use here? I believe both "Save to file" and "Save (minimal)" are both technically permanent saves it just depends on if there is an existing .conf file already or not. @trond-snekvik is that right? We might want to provide a little more context around the saving options for this section.

- Save: saves temporary changes to the <build>/zephyr/.conf file.
- Save to file: saves changes to an existing .conf file.
- Save (minimal): saves the smallest subset of settings to a new .conf file.

Reply:
I would say "permanently" is the right concept, but I don't know if it's the right phrase. It's definitely "non-temporary" at least 😄 Agree with the rest!

Reply:
Hm, on second thoughts, maybe I don't need to describe the UI button here, so I rephrased for simplicity. But I did keep the "Save" in line 228.